src/screens/DiscoveryScreenV2/elements/RecentSearch.js

```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
 import AsyncStorage from '@react-native-async-storage/async-storage';
-import {Keyboard, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import { Keyboard, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
 
 import DiscoveryAction from '../../../context/actions/discoveryAction';
 import GeneralComponentAction from '../../../context/actions/generalComponentAction';
 import IconClear from '../../../assets/icon/IconClear';
 import RecentSearchItems from './RecentSearchItem';
-import {Context} from '../../../context';
-import {RECENT_SEARCH_TERMS} from '../../../utils/cache/constant';
-import {fonts} from '../../../utils/fonts';
-import {COLORS} from '../../../utils/theme';
+import { Context } from '../../../context';
+import { RECENT_SEARCH_TERMS } from '../../../utils/cache/constant';
+import { colors } from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated from `COLORS` to `colors`. This change seems to be consistent with the changes mentioned in the PR description. However, it's important to ensure that all instances of `COLORS` have been replaced with `colors` throughout the entire codebase to maintain consistency and avoid potential errors.
```diff
- import {COLORS} from '../../../utils/theme';
+ import {colors} from '../../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->